.travis.yml

```diff
@@ -1,28 +1,30 @@
 sudo: false
+dist: trusty
 cache: bundler
 language: ruby
 bundler_args: --without debugging
 script: bundle exec rake ci
 rvm:
   - 2.1
   - 2.2
-  - 2.3.0
-  - rbx-2
+  - 2.3
+  - 2.4
+  - jruby-9.1.7.0
+  - jruby-head
   - ruby-head
+  - rubinius-3
 matrix:
-  include:
-    - rvm: jruby
-      env: JRUBY_OPTS='--2.0 --server -Xcompile.invokedynamic=false'
-    - rvm: jruby-head
-      env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
   allow_failures:
-    - rvm: jruby
+    - rvm: jruby-9.1.7.0
     - rvm: jruby-head
-    - rvm: rbx-2
     - rvm: ruby-head
+    - rvm: rubinius-3
   fast_finish: true
 before_install:
-  - gem install bundler
+  - rvm use @global
```

Comment:
Bundler 1.14 is broken w.r.t. the ruby directive in the Gemfile. This is the magic incantation to replace it with 1.13 on Travis CI.